### DISCUSSION TITLE
Looking for help on OpenID authentication

### DISCUSSION THREAD

**Discussion by sudhakarnnh** [ngoduykhanh/PowerDNS-Admin#1551] [General]
Hi,

I look for help on SSO with OpenID connect with keycloak.  I enabled HTTPS with nginx reverse proxy.

Thanks
sudhakar

**Comment by AzorianMatt** [comment]
@sudhakarnnh you won't likely get a lot of response without including a lot more information in your request. It's not really clear what exactly you want help with.
